website/src/components/MacroComponents/MacrobenchMobile/MacrobenchMobile.jsx

```diff
@@ -14,108 +14,213 @@ See the License for the specific language governing permissions and
-                            <span>{data.diff.Right.Metrics.ComponentsCPUTime.vtgate}</span>
-                            <span>{data.diff.Right.Metrics.ComponentsCPUTime.vttablet}</span>
-                            <span>{formatByteForGB(data.diff.Right.Metrics.TotalComponentsMemStatsAllocBytes)}</span>
-                            <span>{formatByteForGB(data.diff.Right.Metrics.ComponentsMemStatsAllocBytes.vtgate)}</span>
-                            <span>{formatByteForGB(data.diff.Right.Metrics.ComponentsMemStatsAllocBytes.vttablet)}</span>
-                        </div>
-                    </SwiperSlide>
-                </Swiper>
-            </div>
-            
+  return (
+    <div className="macrobench__mobile">
+      <div className="macrobench__mobile__header">
+        <h3>{data.type}</h3>
+        <div className="linkQuery">
+          Click{" "}
+          <Link
+            to="/macrobench/queries/compare"
+            state={{
+              data: data,
+              commitHashLeft: commitHashLeft,
+              commitHashRight: commitHashRight,
+            }}
+          >
+            here
+          </Link>{" "}
+          to see the query plans
         </div>
-    );
+      </div>
+      <div className="macrobenchMobile__container flex">
+        <div className="macrobench__Sidebar flex--column">
+          <span>QPS Total</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>QPS Reads</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>QPS Writes</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>QPS Other</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>TPS</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Latency</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Errors</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Reconnects</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Time</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Threads</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Total CPU time</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>CPU time vtgate</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>CPU time vttablet</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Total Allocs bytes</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Allocs bytes vtgate</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Allocs bytes vttablet</span>
+        </div>
```

Comment:
Let's use the same format for the sidebar's text as the one used in the desktop version.